content/08.outlook.md

```diff
@@ -1 +1,41 @@
 ### Outlook
+
+Throughout this perspective we have highlighted various challenges in applying ML methods to rare disease data as well as examples of approaches that address these challenges.
+The low number of samples while significant is not the only roadblock towards application of ML in rare disease data.
+The high dimensionality of the data collected using modern techniques necessitates the management of the curse of dimensionality in creative ways by learning low dimensional representations from the data.
+It further requires leveraging prior knowledge and transfer learning instead of using traditional machiine learning methods for appropriate interpretation of data. 
+Lastly, anyone applying machine learning methods on rare disease data is advised to delve into fundamental techniques that increase confidence and penalize complexity of the resultant models to enhance generalizabilty of ML in rare disease.
+All of the approaches highlighted in this perspective come with certain challenges or inadequacies that breed mistrust in using these powerful techniques in rare disease.
+We believe that the same challenges that are currently outlined as major pitfalls in applying ML to rare disease can be great opportunities for data generation as well as method development in moving the field forward.
+During our journey through the various challenges, we identified two major areas where mindful strategies can immeasurably enhance the power of machine learning in rare disease and move the field forward.
+
+_Emphasis on not just "more n" but "more meaningful n"._
+
+Mindful addition of data is key for powering the next generation of analysis in rare disease data.
+While there are many techniques to collate rare data from different sources, incorrect data generation may hurt the end goal even if it adds to the size of the dataset.
+In our experience collaboration with domain experts have proved to be critical in gaining insight into potential sources of variation in the datasets.
+As an example, in a study of neurofibromatosis type 1 (NF1) some samples were generated using vastly different surgical techniques (laser ablation and excision vs standard excision). [@doi:10.3390/genes11020226] 
+This resulted in surfacing of substantial biological differences that are a consequence of process (e.g. activation of heat shock protein related pathways), not disease related biology. 
```

Comment:
We didn't specifically interrogate this in this paper, I think, or am I misremembering?

Reply:
e.g. I think we had this hypothesis but didn't follow this particular thread (primarily because we don't have non-laser resected cNF tumor expression data to explore this difference). I think if someone looked for this in the cited paper, they wouldn't find it...but again, could be misremembering.

Reply:
You are correct... we had this hypothesis but did not interrogate it systematically in the paper. As I am re-reading this, I see that my phrasing suggests more than what I intended to say. Will rephrase :)